src/Handler/Workspace/ResetPassword/ResetWorkspacePasswordHandler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Workspace\ResetPassword;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Workspace\ResetWorkspacePasswordCommand;
+use Keboola\StorageDriver\Command\Workspace\ResetWorkspacePasswordResponse;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Shared\Utils\Password;
+use Keboola\StorageDriver\Teradata\TeradataSessionManager;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+final class ResetWorkspacePasswordHandler implements DriverCommandHandlerInterface
+{
+    private TeradataSessionManager $manager;
+
+    public function __construct(TeradataSessionManager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     * @param ResetWorkspacePasswordCommand $command
+     */
+    public function __invoke(
+        Message $credentials, // project credentials
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof ResetWorkspacePasswordCommand);
+
+        // validate
+        assert(!empty($command->getWorkspaceUserName()), 'ResetWorkspacePasswordCommand.workspaceUserName is required');
+
+        $db = $this->manager->createSession($credentials);
+
+        $newWsPassword = Password::generate(
+            30,
+            Password::SET_LOWERCASE | Password::SET_UPPERCASE | Password::SET_NUMBER | Password::SET_SPECIAL_CHARACTERS
+        );
+
+        $db->executeStatement(sprintf(
+            'MODIFY USER %s AS PASSWORD = %s FOR USER;',
+            TeradataQuote::quoteSingleIdentifier($command->getWorkspaceUserName()),
+            TeradataQuote::quoteSingleIdentifier($newWsPassword)
+        ));
+
```

Comment:
Ještě sejmi existující sessions přes `SELECT SYSLIB.AbortSessions (-1, '<user>', 0, 'Y', 'Y');`

Reply:
Nevím jestli to půjde nějak adekvátně otestovat, ale že to nespadne je asi good enough

Reply:
OK, zkusim

Reply:
tak to chce nejaky dalsi prava :-/
> [Teradata][ODBC Teradata Driver][Teradata Database](-3523)The user does not have EXECUTE FUNCTION access to SYSLIB.AbortSessions. (SQLExecDirect[4294963773] at /usr/src/php/ext/pdo_odbc/odbc_driver.c:246)

Reply:
Hmm tak to bude ten projektový uživatel potřebovat.

Reply:
jj, zkousim to tam pridat...

Reply:
✅  pridano

Reply:
Jeste to potom pridam do testu...